src/lib/stitching/gravity/schema.ts

```diff
@@ -98,5 +108,14 @@ export const executableGravitySchema = () => {
         return name
       }
     }),
+    new FilterRootFields((operation, name) => {
```

Comment:
This is new, we want to start excluding certain mutations from Gravity's schema.